mockingbird-compiler/src/main/kotlin/com/careem/mockingbird/ProjectExplorer.kt

```diff
@@ -70,13 +65,8 @@ class ProjectExplorer {
 
     private fun Project.traverseProjectTree() {
         this.subprojects.forEach {
-            try {
-                moduleMap[it.name] = it
-                it.traverseProjectTree()
-            } catch (udo: org.gradle.api.UnknownDomainObjectException) {
-                this@ProjectExplorer.logger.warn("${it.name} -> SKIPPED")
-            }
```

Comment:
I don't think this will work when you build sample

Reply:
it passed actually, this `gw clean && gw generateMocks && gw build` commend passed
the issue we saw yesterday is nothing related to this try-catch block, it's something else

Reply:
put it back as discussed offline